.github/workflows/windows.yml

```
@@ -13,15 +13,16 @@ defaults:
 jobs:
   build:
     runs-on: ${{ matrix.runs-on }}
-    name: ${{ matrix.sys.compiler }}-${{ matrix.build-system }}-${{ matrix.config.name }}
+    name: ${{ matrix.sys.compiler }} / ${{ matrix.build-system }} / ${{ matrix.config.name }} / date-polyfill ${{ matrix.sys.date-polyfill}}
     strategy:
       fail-fast: false
       matrix:
         runs-on: [windows-latest]
         sys:
-        - {compiler: default}
-        - {compiler: msvc}
-        - {compiler: clang}
+        - {compiler: msvc, date-polyfill: 'ON' }
```

FYI, I am subscribed to GitHub Action Runners' new releases.
```suggestion
        # TODO: Reintroduce default (or just mingw) when the new version is release
        # See: https://github.com/actions/runner-images/commits/main/images/windows/Windows2022-Readme.md
        # See: https://www.mingw-w64.org/changelog/
        - {compiler: msvc, date-polyfill: 'ON' }
```

---

I suggest we explicitely invoke mingw rather than calling it because it is the default toolchain with Ninja.